docs/deprecated.md

```diff
@@ -19,6 +19,19 @@ The following list of features are deprecated in Engine.
 To learn more about Docker Engine's deprecation policy,
 see [Feature Deprecation Policy](https://docs.docker.com/engine/#feature-deprecation-policy).
 
+### Pushing and pulling with image manifest v2 schema 1
+
+**Deprecated in Release: v19.03.0**
+**Will be removed in Release: v19.09.0**
```

Comment:
```suggestion

**Target For Removal In Release: v19.09.0**
```

(also needs the empty line before)

Reply:
added

Reply:
I think something went wrong @andrewhsu 😅 
The suggestion has not been fully applied.